examples/get_async_bleak.py

```diff
@@ -7,9 +7,12 @@
 
 
 async def main():
-    async for data in RuuviTagSensor.get_data_async():
-        print(data)
+    try:
+        async for data in RuuviTagSensor.get_data_async():
+            print(data)
+    except asyncio.exceptions.CancelledError:
+        print("Scan stopped")
 
 
 if __name__ == "__main__":
-    asyncio.get_event_loop().run_until_complete(main())
+    asyncio.run(main())
```

Comment:
We could add this kind of comment to examples.
```py
if __name__ == "__main__":
    # Note: Python 3.10 or later is required with asyncio.run
    # For older versions of Python, use asyncio.get_event_loop().run_until_complete(main())
    asyncio.run(main())
```

Reply:
Sure, added.